crates/wdl-lint/src/tags.rs

```diff
@@ -33,8 +33,11 @@ pub enum Tag {
     /// Rules associated with the use of deprecated language constructs.
     Deprecated,
 
-    /// Rules associeted with documentation.
+    /// Rules associated with documentation.
     Documentation,
+
+    /// Rules associated with compatibility with documentation rendering.
+    DocRendering,
```

Comment:
I do not love this new tag, but I do not think that this rule should be part of the default set of rules, which includes the `Documentation` tag. Open to hearing alternatives here